api/src/main/java/org/openmrs/api/db/DelegatingFullTextSession.java

```diff
@@ -134,10 +161,4 @@ public MassIndexer createIndexer(Class<?>... types) {
 	public FullTextSharedSessionBuilder sessionWithOptions() {
 		return delegate.sessionWithOptions();
 	}
-
-	@Override
-	public FullTextQuery createFullTextQuery(QueryDescriptor descriptor, Class<?>... entities) {
-		throw new UnsupportedOperationException();
-	}
-	
```

Comment:
I'm not sure why this is included in this commit?

Reply:
this was among the new commits that were introduced, let me fix everything